var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -70,8 +70,8 @@ def cmake_args(self):
         options = list()
 
         options.extend([
-            "-DCMAKE_INSTALL_PREFIX=%s" % prefix,
-            "-DCMAKE_INSTALL_NAME_DIR:PATH=%s/lib" % prefix,
+            "-DCMAKE_INSTALL_PREFIX=%s" % self.prefix,
```

Comment:
This is already set in the `_std_args` of `CMakePackage` so you don't actually need it.

Reply:
Even during the module creation? If that's the case I'm honestly not sure what I changed to get it to work then.

Reply:
I'm not sure why either of these lines are present in the file, but you're correct that just using `prefix` without the `self.` is a bug.

Reply:
In that case should I close this and open up an issue? I'm not sure I have any other examples where I've experienced this.

Reply:
It looks like these lines were added back in #9489. @luszczek do you recall why these lines were necessary?

Reply:
I copied it from version 17 when PLASMA was installed with Makefile. I removed it and tested on x86 and POWER with OpenBLAS and Netlib-LAPACK. It is not necessary.